build/Common.props

```diff
@@ -41,7 +41,7 @@
     <StackExchangeRedisPkgVer>[2.1.58,3.0)</StackExchangeRedisPkgVer>
     <StyleCopAnalyzersPkgVer>[1.1.118,2.0)</StyleCopAnalyzersPkgVer>
     <SystemCollectionsImmutablePkgVer>[1.4.0,6.0)</SystemCollectionsImmutablePkgVer>
-    <SystemDiagnosticSourcePkgVer>[5.0.0,6.0)</SystemDiagnosticSourcePkgVer>
+    <SystemDiagnosticSourcePkgVer>5.0.0</SystemDiagnosticSourcePkgVer>
```

Comment:
Hey, I know this is the opposite of what this PR is supposed to achieve, but have you considered setting the minimum version for DiagnosticSource to 5.0.1? We were recently burned by the timer resolution bug in 5.0.0, and I'm sure other teams will stumble over that.

Reply:
Since this repo requires only 5.0.0, I think this is best left to customers.
It'd be a lot of maintenance work, to keep updating/releasing whenever patch versions are released in DS package.

Reply:
I kind of agree with @johnduhart here. We know 5.0.0 is kind of flawed so we should help out our users and force them onto 5.0.1. DS is a special package because we work so closely with that team 😄

Reply:
Even if we do this one time, we won't be able to guarantee to do this everytime there is a new patch, right?

Reply:
Will come back to this topic. (Need to do similar relaxation for other packages from /dotnet/runtime repo)